tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot

```diff
@@ -29,7 +29,7 @@ Basic docker pull, restart, and remove image
     # Gather logs before rebooting
     Run Keyword And Continue On Failure  Gather Logs From Test Server  -before-reboot-1
     Reboot VM  %{VCH-NAME}
-    Wait For VCH Initialization  20x  5 seconds
+    Wait For VCH Initialization  30x  10 seconds
```

Comment:
These looong wait times - I can see why we end up with them but it would be good to have some cross build tracking and anomaly detection. The VCH should come up _fast_ with the main variables being:
* dhcp response time
* vSphere infrastructure response time
* number of existing images/containers for the VCH.
* vSphere incorrect host routing in HA cases

The first 3 of these should be under our control so a much more predictable time interval for restart will allow us to catch incorrect behaviours. The latter _should_ be occurring only in specific tests and even then we should be getting vSphere to improve it.

When I say fast I'm talking on the order of 20seconds with a small number of images (which is all these tests ever have). Time to return to function after reboot *should* be a performance metric for us _and_ a quality measure.

Reply:
@mhagen-vmware ?

Reply:
We don't currently have any performance tests within this infra, and we aren't going to add them in this PR.  We have 2 options: 1. have our perf folks actually test reboot (which I don't think they do), 2. create a story and we can do it in a future PR.